SourceLink.Create.CommandLine/SourceLink.Create.CommandLine.targets

```diff
@@ -18,13 +18,13 @@
   </PropertyGroup>
 
   <Target Name="SourceLinkCreate">
-    <Exec Condition="'$(SourceLinkRootDirectory)' == ''" Command="$(SourceLinkRootDirectoryCommand)" ConsoleToMsBuild="true">
+    <Exec EchoOff="true" Condition="'$(SourceLinkRootDirectory)' == ''" Command="$(SourceLinkRootDirectoryCommand)" ConsoleToMsBuild="true">
```

Comment:
Use `StandardOutputImportance="Low"` instead. I'll explain more in https://github.com/Microsoft/msbuild/issues/2826.